src/adr/0014-blobstore-storage-policies.md

```diff
@@ -35,8 +35,6 @@ The proposed policies include:
  - LowCostStoragePolicy: The blob is expected to be saved in low cost storage. Access is expected to be unfrequent.
  - PerformantStoragePolicy: The blob is expected to be saved in performant storage. Access is expected to be frequent.
 
-An HybridBlobStore will replace current UnionBlobStore and will allow to choose between Cassandra and ObjectStorage implementations depending on the policies.
```

Comment:
I'm not sure we should make that disappear from an ADR previously written, but instead create a new ADR that supersedes this one? @chibenwa

Reply:
Yes don't alter previously approuved ADRs. Removal is documented in 0025